src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java

```diff
@@ -480,8 +488,17 @@ public ModelAndView verification(@ModelAttribute("authorizeCommand") @Validated
 
         final boolean validationErrors = bindingResult.hasErrors();
         if (validationErrors) {
-            if (StringUtils.isEmpty(request.getCode())) {
+            final List<FieldError> codeErrors = ofNullable(bindingResult.getFieldErrors("code"))
```

Comment:
Why use ofNullable here? Seems unnecessary

Reply:
just to be safe in case the variable changes then it just breaks validation and not the entire feature